docs/source/datasets.rst

```diff
@@ -33,7 +32,101 @@ to adhere to a certain format. ASReview accepts the following formats:
    be comma separated and UTF-8 encoded. For CSV files, the software accepts a set of 
    predetermined labels in line with the ones used in RIS files. 
 
-
+Metadata
```

Comment:
Maybe group the following 3 sections (Citation managers, search engines, sysrev software) into one group. Not sure about this.

Reply:
Maybe we could call them Export tools?